Server/htdocs/AppController/commands_RSM/utilities/RSMverifyBody.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+//***************************************************************************************
+// Structure
+//***************************************************************************************
+
+function checkBodyIsJsonObject($body)
+{
+  global $RSallowDebug;
+  if (!is_object($body)) {
+    if ($RSallowDebug) returnJsonMessage(400, "Request body must be a JSON object '{}'");
+    else returnJsonMessage(400, "");
+  }
+}
+
+function checkIsJsonObjectInArray($body)
+{
+  global $RSallowDebug;
+  if (!is_object($body)) {
+    if ($RSallowDebug) returnJsonMessage(400, "Request body array elements must be JSON objects '{}'");
+    else returnJsonMessage(400, "");
+  }
+}
+
+function checkIDsIsArray($body)
+{
+  global $RSallowDebug;
+  if (isset($body->IDs)) {
+    if (!is_array($body->IDs)) {
+      if ($RSallowDebug) returnJsonMessage(400, "Request body 'IDs' field must be an array");
+      else returnJsonMessage(400, "");
+    }
+  }
+}
+
+function checkPropertyIDsIsArray($body)
+{
+  global $RSallowDebug;
+  if (isset($body->propertyIDs)) {
+    if (!is_array($body->propertyIDs)) {
+      if ($RSallowDebug) returnJsonMessage(400, "Request body 'propertyIDs' field must be an array");
+      else returnJsonMessage(400, "");
+    }
+  }
+}
+
+function checkBodyIsArray($body)
+{
+  global $RSallowDebug;
+  if (!is_array($body)) {
+    if ($RSallowDebug) returnJsonMessage(400, "Request body must be an array '[]'");
+    else returnJsonMessage(400, "");
+  }
+}
```

Comment:
This function name should be checkIsArray instead, as it can check anything (not just a body)